### ISSUE TITLE
Set up weekly meetings with Dan Perkins to discuss screen readers 

### ISSUE THREAD

**Issue by hatfieldjm4** [GSA/EDX#272]
To better prioritize the AMP and Lighthouse outputs, Mark and Jonah should meet with Dan Perkins to better understand what really matters to people who rely on screen readers.

**Comment by hatfieldjm4** [comment]
There is a Windows-native tool called Narrator that we all should have access to where we can hear what a screen reader provides.
